test/oli_web/live/collaboration_live_test.exs

```diff
@@ -317,6 +317,24 @@ defmodule OliWeb.CollaborationLiveTest do
     end
   end
 
+  describe "user can access when is logged in as a system admin" do
+    setup [:admin_conn, :create_project_and_section]
+
+    test "returns the collab space config", %{
+      conn: conn,
+      admin: admin,
+      project: project,
+      page_revision: page_revision
+    } do
+      conn =
+        conn
+        |> Pow.Plug.assign_current_user(admin, OliWeb.Pow.PowHelpers.get_pow_config(:author))
+        |> get(live_view_author_edit(project.slug, page_revision.slug))
```

Comment:
We should test that an admin can see the collab space accessing the preview route instead of this one